angular1/README.md

```diff
@@ -0,0 +1,28 @@
+# Angular Seed TypeScript
+This is based on https://github.com/angular/angular-seed. If you are curious about how the conversion of the JS project was done to the TS project checkout [CONVERSION.md](./CONVERSION.md).
+
+## Running
+The following are specific to TypeScript
+
+Setup TypeScript:
+```bash
+npm install typescript -g
+npm install tsd -g
+```
+Start the TypeScript compiler in watch mode (either in the `app` folder or in the `e2e-tests` folder) and **leave it running**:
+
+```bash
+# For app
+tsc --watch --p app
+# For e2e-tests
+tsc --watch --p e2e-tests
+```
+
+That's it. You have typescript setup and ready to go. You can follow the steps of JavaScript ([README-JS](./README-JS.md)) from this point on in a new window starting at the install dependencies section.
```

Comment:
Nit: You can, then, follow the standard steps for seeded AngularJS web app in a new window, using [README-JS](./README-JS.md) starting at the section, [Install Dependencies](https://github.com/basarat/TypeScriptSamples/blob/angular1/angular1/README-JS.md#install-dependencies).